rust/client-ffi/Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
-name = "android-client-ffi"
-# mark:next-android-version
-version = "1.5.2"
+name = "client-ffi"
+version = "0.1.0"
 edition = { workspace = true }
 license = { workspace = true }
 
 [lib]
 name = "connlib"
-crate-type = ["lib", "cdylib"]
+crate-type = ["lib", "cdylib", "staticlib"]
```

Comment:
Will be interesting to see how much larger the release build is.

Reply:
`staticlib` will only be needed once we use this for Swift.